Makefile

```diff
@@ -1,10 +1,27 @@
-GO ?= $(shell command -v go 2> /dev/null)
 NPM ?= $(shell command -v npm 2> /dev/null)
 CURL ?= $(shell command -v curl 2> /dev/null)
 MANIFEST_FILE ?= plugin.json
 MM_UTILITIES_DIR ?= ../mattermost-utilities
 
+MINIMUM_SUPPORTED_GO_MAJOR_VERSION = 1
+MINIMUM_SUPPORTED_GO_MINOR_VERSION = 12
 export GO111MODULE=on
+GO ?= $(shell command -v go 2> /dev/null)
+GOFLAGS ?= $(GOFLAGS:)
+BUILD_DATE = $(shell date -u)
+BUILD_HASH = $(shell git rev-parse HEAD)
+BUILD_HASH_SHORT = $(shell git rev-parse --short HEAD)
+LDFLAGS += -X "main.BuildDate=$(BUILD_DATE)"
+LDFLAGS += -X "main.BuildHash=$(BUILD_HASH)"
+LDFLAGS += -X "main.BuildHashShort=$(BUILD_HASH_SHORT)"
```

Comment:
Ah, yah, this is much better. Thank you!
I should have thought of that... <facepalm>